libs/core/src/lib/info-label/info-label.component.ts

```diff
@@ -0,0 +1,69 @@
+import { Component, ViewEncapsulation, ChangeDetectionStrategy, Input, OnInit, ElementRef, OnChanges } from '@angular/core';
+import { CssClassBuilder, applyCssClass } from '../utils/public_api';
+
+type LabelType = 'numeric' | 'only-icon' | 'icon';
+
+@Component({
+    selector: 'fd-info-label',
+    templateUrl: './info-label.component.html',
+    styleUrls: ['./info-label.component.scss'],
+    encapsulation: ViewEncapsulation.None,
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class InfoLabelComponent implements OnInit, OnChanges, CssClassBuilder {
+
+    /** Select type defines the label types */
```

Comment:
The TypeDocs should mention what are these types. Whatever you put here appears in the API documentation of the library, so it's important for the user to know what are the available types.